content/en/docs/components/katib/experiment.md

```diff
@@ -41,159 +42,166 @@ the experiment from the UI or from the command line.
 
 These are the fields in the experiment configuration spec:
 
-* **parameters**: The range of the hyperparameters or other parameters that you 
-  want to tune for your ML model. The parameters define the *search space*,
-  also known as the *feasible set* or the *solution space*.
-  In this section of the spec, you define the name and the distribution 
+- **parameters**: The range of the hyperparameters or other parameters that you
+  want to tune for your ML model. The parameters define the _search space_,
```

Comment:
Please double check that "ML" has been predefined in this doc (it's a small thing and we know it's "machine learning" (what else!) but this makes the guide more "proper" :))

Reply:
Sure, I explain abbreviation.